packages/supi/src/install/index.ts

```diff
@@ -401,6 +403,41 @@ export async function mutateModules (
       })
     }
 
+    if (maybeOpts.strict) {
+      const project = projectsToInstall[0]
```

Comment:
I think each project should be checked in a loop and an exception should be thrown only if none of the projects have some of the dependencies

Reply:
I am doing it: in L412 I iterate all projects to get a list of all dependencies then iterate through inputs to check if each has wantedDependency